src/org/osm2world/core/world/modules/ExternalModelModule.java

```diff
@@ -97,13 +98,28 @@ private ExternalModelNodeWorldObject(MapNode node, Model model) {
 		public Iterable<EleConnector> getEleConnectors() {
 		
 			if (eleConnector == null) {
-				VectorXZ pos = element.getPos();
+				VectorXZ pos = getPosition(element);
 				eleConnector = new EleConnector(pos, element, GroundState.ON);
 			}
 			
 			return singletonList(eleConnector);
 			
 		}
+
+		private VectorXZ getPosition(MapNode element) {
```

Comment:
@tordanik this is the part where I've stucked for now. Usually model 0. 0 point doesn't match mapNode position. So I have to calculate origin transform vector to move it. I need an access to mapProjection, but not sure waht would be the gracefull way to do that. Not sure that it's worth to add a reference on it in mapNode.